### DISCUSSION TITLE
Does this have to be ALL ALBUMS ?

### DISCUSSION THREAD

**Discussion by Nugroho222** [TheLastGimbus/GooglePhotosTakeoutHelper#334] [I NEED HELP]
I tried to use this to repair the timestamps of the takeout files. I only wanted to repair some albums, not all albums as it will be too big. But it always failed. Does it have to be ALL ALBUMS, as the Readme mentioned "do not deselect albums". Can I change the name of the takeout folders in order to use this ? Thanks

**Comment by TheLastGimbus** [comment]
in general, gpth is made to work only with all-at-once

buuut it think you should be fine by selecting all year folders and then only albums that you want

**Reply by Nugroho222** [reply]
Thanks for the answer. But I don't understand how to "select all year folders". Could you elaborate ?

**Reply by TheLastGimbus** [reply]
while selecting which albums you want in takeout page

**Comment by Nugroho222** [comment]
Dear Mateus,
Please elaborate about the selection of all year folder. I really don't understand what you mean. 
I need it very much. 
Thanks,Edi. 
 
 
  On Fri, Aug 9, 2024 at 1:01, Mateusz ***@***.***> wrote:   


while selecting which albums you want in takeout page
